app/forms/idv/api_document_verification_form.rb

```diff
@@ -95,9 +95,13 @@ def throttle_if_rate_limited
 
     def throttled_else_increment
       return unless document_capture_session
-      @throttled = Throttler::IsThrottledElseIncrement.call(
-        document_capture_session.user_id,
-        :idv_acuant,
+      @throttled = throttler.throttled_else_increment?
+    end
+
+    def throttler
```

Comment:
In other classes, this is called `throttle`. Should it be the same here?

Reply:
yup, I had a name change mid-PR, forgot to clean these up, thanks, fixed in a07b5cc86fe3facdb8d814aff5079f6a939e6c27